src/i18n/fr/tester.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  page: {
+    label: 'Testeur De Clavier'
+  },
+  keycodeStatus: {
+    label: 'Touches Détectées'
+  },
+  back: {
+    label: 'Retour',
+    title: 'Retour au configurateur'
+  },
+  reset: {
+    label: 'Réinitialiser',
+    title: 'Réinitialise les touches détectées'
+  },
+  letters: {
+    keycode: {
+      label: 'code de la touche'
+    },
+    code: {
+      label: 'code'
+    },
+    key: {
+      label: 'touche'
+    }
+  },
+  docs: {
+    paragraph:
+      'Note: La touche et le code détecté.es peuvent varier selon votre localisation. Docs'
+  },
+  chatter: {
+    label: 'Seuil de détection d\'activations répétées (ms)',
+    detectedAlert: 'Activations répétées détectées'
```

Comment:
Mention of `Chatter` would be a good addition like:
`Activations répétées détectées (Chatter)`
Because it's the universal way to call it.

Reply:
I agree, good idea thank you